src/config/theme.ts

```diff
@@ -341,6 +345,9 @@ const createThemeOptions = (theme: Theme) => ({
                 theme.palette.alerts.low.header
               ),
             },
+            '& .MuiAlert-deleteIcon': {
+              color: theme.palette.alerts.low.icon || 'rgba(0, 0, 0, 0.54)',
```

Comment:
Is this needed? I don't see any difference when disabling this, it looks like this is the mui default anyway? If it's just slightly different, I would prefer removing this to keep the theme simpler.

Reply:
The `'rgba(0, 0, 0, 0.54)'` color here is indeed the mui default. I added the icon color as an option on the theme in order to get the effect specified by Ash in figma ([here](https://www.figma.com/file/F9EHR2IZYuMAlOJ9aOG2IL/OP-%2F-HMIS-%2F-Phase-2-(2024)?type=design&node-id=622%3A2869&mode=design&t=ihkylfJD9cxM7VpK-1)) where the high priority icon is a different color from the others, for better contrast against the red background.

I think we could do away with it if we use `getContrastColor` (but then the icons would be black/white instead of this nice grey), or we could go back to using a paler red for high priority alerts (we switched to this one in order to be consistent with `theme.palette.error.main`).

Reply:
Ah gotcha, thanks. 

So these class overrides are not needed for `low` and `medium` colors, so we can remove those from the theme.

For the high priority, I would just do specify a rule here saying that high-priority alerts should use this color for icon buttons inside the dark-colored header:
```
            '.MuiAlertTitle-root .MuiIconButton-root': {
              color: theme.palette.alerts.high.icon,
            },
```
that way we don't need to be passing `className`s around. Does that make sense?

Reply:
Yes that's much better!